proposals/nnnn-alert-icon.md

```diff
@@ -0,0 +1,147 @@
+```
+
+### Mobile API additions
+
+```xml
+<enum name="ImageFieldName">
+  :
+  <element name="alertIcon">
+    <description>The image field for Alert</description>>
+  </element>
+</enum>>
+<function name="Alert" functionID="AlertID" messagetype="request">
+   :
+   <param name="alertIcon" type="Image" mandatory="false" >
+     <description>
+       Image struct determining whether static or dynamic icon.
+       If omitted on supported displays, no (or the default if applicable) icon should be displayed.
+     </description>
+  </param>
+</function>
+```
+
+### SDL Android
+The Android library would need to add `setAlertIcon()` and `getAlertIcon()` method in `Alert.java`
+```java
+public class Alert extends RPCRequest {
+  public static final String KEY_PLAY_TONE = "playTone";
+  :
+  public static final String KEY_SOFT_BUTTONS = "softButtons";
+  public static final String KEY_ICON = "icon";   // <--- add
+
+  :
+
+  /**
+   * <p>Sets the Image
+   * If provided, defines the image to be shown along with an alert</p>
+   * @param alertIcon
+   *            <p>an Image obj representing the Image obj shown along with an
+   *            alert</p>
+   *            <p>
+   *            <b>Notes: </b>If omitted on supported displays, no (or the
+   *            default if applicable) icon will be displayed</p>
+   */
+  public void setAlertIcon(Image alertIcon) {
+    setParameters(KEY_ICON, alertIcon);
+  }
+
+  /**
+   * <p>Gets the image to be shown along with an alert </p>
+   * 
+   * @return Image -an Image object
+   */
+  public Image getAlertIcon() {
+    return (Image) getObject(Image.class, KEY_ICON);
+  }
+}
+```
+
+### SDL iOS
+The iOS library would need a new constant `SDLNameAlertIcon`, add new property `alertIcon`, and create corresponding `setAlertIcon` and `getAlertIcon` functions in `SDLAlert.m`
+
+**SDLNames.h**
+```objectivec
+extern SDLName const SDLNameAlertIcon;
+```
+**SDLNames.m**
+```objectivec
+SDLName const SDLNameAlertIcon = @"alertIcon"
+```
+**SDLImageFieldName.m**
+```objectivec
+SDLImageFieldName const SDLImageFieldNameAlertIcon = @"alertIcon"
+```
+**SDLAlert.h**
+```objectivec
+/**
+ * @abstract Image struct containing a static or dynamic icon
+ *
+ * @discussion If provided, defines the image to be be shown along with an alert
+ * 
+ * If omitted on supported displays, no (or the default if applicable) icon will be displayed
+ *
+ * Optional
+ */
+@property (nullable, strong, nonatomic) SDLImage *alertIcon;
+```
+**SDLAlert.m**
+```objectivec
+#import "SDLAlert.h"
+#import "SDLImage.h"
+#import "SDLName.h"
+
+- (void)setAlertIcon:(nullable SDLImage*)alertIcon {
+        [parameters setObject:alertIcon forName:SDLNameAlertIcon];
+}
+
+- (nullable SDLImage *)alertIcon {
+    return [parameters sdl_objectForName:SDLNameAlertIcon ofClass:SDLImage.class];
+}
+```
+
+## Potential downsides
+
+The proposed change is backward compatible and will not cause a breaking change. However the API of `Alert` are inconsistent.
```

Comment:
Please update to "...is inconsistent."

Reply:
@theresalech fixed in commit [53a1e0c](https://github.com/smartdevicelink/sdl_evolution/pull/513/commits/53a1e0c9fe99af35cbbf34ff221222460c8280d7).